src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -56,4 +51,11 @@ await dispatcher.Dispatch(new TransportOperations(operation), transportTransacti
 
     static ILog log = LogManager.GetLogger<NServiceBusMetricReport>();
     TransportTransaction transportTransaction = new TransportTransaction();
-}
+
+    static Dictionary<string, string> headers = new Dictionary<string, string>
+    {
+        { Headers.EnclosedMessageTypes, "NServiceBus.Metrics.MetricReport" }, // without assembly name to allow ducktyping
+        { Headers.ContentType, ContentTypes.Json },
+        { Headers.MessageIntent, MessageIntentEnum.Send.ToString() }
```

Comment:
In Heartbeat and SagaAudit we set it as well. It is not strictly required since the core will fallback to messageIntent = default(MessageIntentEnum) which is Send